exchanges/request/report.go

```diff
@@ -0,0 +1,27 @@
+package request
+
+import (
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+type Reporter interface {
+	Latency(name, method, path string, t time.Duration)
```

Comment:
Can we add in a verbose param? This is so we can omit the basic reporter output and it won't really affect any custom reporters or the global if not defined that way.

Reply:
sounds good, i ended up doing the same thing but slightly different, looked a bit weird to pass a verbose flag to the interface method just for the sake of `DefaultReporter`